package.json

```diff
@@ -17,7 +17,9 @@
     "verbose": true
   },
   "dependencies": {
+    "@micromodal/repo": "https://github.com/andreyvolokitin/Micromodal.git#patch-1",
```

Comment:
I should have said we should fork the branch to made by many. A branch might change and include bad code in the future.

Reply:
Good point, I'll do that. I was thinking about that possibility, but didn't think about forking it ourselves.